### ISSUE TITLE
[Feature]: 请问如何对Llama做Example？

### ISSUE THREAD

**Issue by isaac1216** [AntCPLab/OpenBumbleBee#9]
### Feature Request Type

Build/Install

### Have you searched existing issues?

Yes

### Is your feature request related to a problem?

非常感谢代码的开源！我根据源码复现了BERT、GPT、ViT的隐私2PC推理，但是我看论文里面也对Llama-7B做了测试，源码里并没有Llama Example部分，请问如果我想将这套源码放到Llama上面需要做哪些改动？

### Describe features you want to add to SPU

None.

### Describe features you want to add to SPU

None.

**Comment by Fennel1** [comment]
您好！想问一下您复现的隐私2PC推理，CPU与SPU之间的输出差异大吗？下面是我复现VIT的结果，logits差异还是蛮大的。导致我跑imagenet1k的测试结果cpu(top-1 75%)和spu(top-1 36%)之间差距很大。想问一下有没有遇到相似的情况。
```Running on CPU ...
CPU runtime: 0.9341721534729004s
Top 5 logits  [[-0.26889285  0.82709914 -0.07265731  0.42345694  0.56375045]]
CPU Predicted class: Egyptian cat
Running on SPU ...
SPU runtime: 254.91946721076965s
Top 5 logits  [[-0.23344421  0.8664665  -0.07375717  0.43945694  0.6165848 ]]
SPU Predicted class: Egyptian cat```

**Comment by fionser** [comment]
@Fennel1 logits 有一定差别。因为很多 SPU 上的计算是近似的。但是我自己测的 image1k top1 的区别不是很大。

Update: 请 Pull 当前 ndss 的代码；已经更新的 ViT 上的 gelu 调用逻辑

似乎是 ViT 的 Gelu 比较敏感。
可以把这[一行](https://github.com/AntCPLab/OpenBumbleBee/blob/ndss/libspu/kernel/hal/intrinsic/nn/activation.cc#L170)
改掉。

```cpp
auto target_field = FieldType::FM32;
```
```cpp
auto target_field = src_field;
```

改前 `n0144764_tench.JPEG`
```
Running on CPU ...
CPU runtime: 0.4367516040802002s
Top 5 logits  [[14.794788    6.9146495  -0.50997573  0.8706742  -0.21364349]]
CPU Predicted class: tench, Tinca tinca
Running on SPU ...
SPU runtime: 173.60636353492737s
Top 5 logits  [[ 2.526207    1.226181   -0.02405548  0.54080963  0.55999756]]
SPU Predicted class: tench, Tinca tinca
```

改后
```
CPU runtime: 0.4412968158721924s
Top 5 logits  [[14.794788    6.9146495  -0.50997573  0.8706742  -0.21364349]]
CPU Predicted class: tench, Tinca tinca
Running on SPU ...
SPU runtime: 184.45504927635193s
Top 5 logits  [[14.7757      6.779152   -0.52960205  0.8624573  -0.235363  ]]
SPU Predicted class: tench, Tinca tinca
``` 

```
Running on CPU ...
CPU runtime: 0.44779181480407715s
Top 5 logits  [[-0.47618812 -0.6702112  -0.20885047 -0.37564492 -0.3149698 ]]
CPU Predicted class: robin, American robin, Turdus migratorius
Running on SPU ...
SPU runtime: 179.64360451698303s
Top 5 logits  [[-0.4501152  -0.6354294  -0.271904   -0.39747238 -0.29481125]]
SPU Predicted class: robin, American robin, Turdus migratorius
```

**Comment by fionser** [comment]
@isaac1216 按照 PUMA 论文的例子。把 Python 的脚本拿过来就可以（对应改好后端为 CHEETAH）https://github.com/secretflow/spu/tree/main/examples/python/ml/flax_llama7b
